src/main/resources/db/migration/V1.9__add_unique_publication_index.sql

```diff
@@ -0,0 +1,3 @@
+CREATE UNIQUE INDEX IF NOT EXISTS unique_publication_index
```

Comment:
Could you add a brief comment in this file to explain why this is added

Reply:
Have added a comment to the file